tests/types/src/valid.ts

```diff
@@ -3184,6 +3184,13 @@ const paymentRequest: PaymentRequest = stripe.paymentRequest({
   },
 });
 
+const paymentRequestOBO: PaymentRequest = stripe.paymentRequest({
+  country: 'US',
+  currency: 'usd',
```

Comment:
Can we also add a test in invalid.ts that checks that you cannot update onBehalfOf?